packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -183,29 +183,46 @@ export default class MainProcess {
   private _setAppMenu() {
     const isMac = this.settings.platform === 'darwin';
 
-    const template: MenuItemConstructorOptions[] = [
-      ...(isMac ? ([{ role: 'appMenu' }] as const) : []),
-      ...(isMac ? [] : ([{ role: 'fileMenu' }] as const)),
+    const macTemplate: MenuItemConstructorOptions[] = [
+      { role: 'appMenu' },
       { role: 'editMenu' },
       { role: 'viewMenu' },
-      isMac
-        ? { role: 'windowMenu' }
-        : {
-            label: 'Window',
-            submenu: [{ role: 'minimize' }, { role: 'zoom' }],
+      {
+        label: 'Window',
+        submenu: [{ role: 'minimize' }, { role: 'zoom' }],
+      },
+      {
+        role: 'help',
+        submenu: [
+          {
+            label: 'Learn More',
+            click: () => {}, // TODO: add link to Connect docs
```

Comment:
Haha okay, I was wondering why this button doesn't do anything when I was testing out the Windows build last week.